eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -674,11 +666,22 @@ class EraGov era => ConwayEraGov era where
   drepPulsingStateGovStateL :: Lens' (GovState era) (DRepPulsingState era)
   committeeGovStateL :: Lens' (GovState era) (StrictMaybe (Committee era))
 
+  getDRepDistr :: GovState era -> Map (DRep (EraCrypto era)) (CompactForm Coin)
+  getConstitution :: GovState era -> Constitution era
```

Comment:
Why keep these functions here, `getConstitution` and `getCommitteeMembers` ?  can't we use the lenses that already exist in this class, `constitutionGovStateL` and `committeeGovStateL` ?    (and probably the same for  `getDRepDistr`) 
Or is there some restriction about not using lenses in the queries where they are used?

Reply:
That's an interesting question. I think I agree with you. :+1: